stdlib/2/__builtin__.pyi

```diff
@@ -736,6 +736,15 @@ def divmod(a: int, b: int) -> Tuple[int, int]: ...
 def divmod(a: float, b: float) -> Tuple[float, float]: ...
 def exit(code: Any = ...) -> NoReturn: ...
 @overload
+def filter(function: Callable[[AnyStr], Any],
+           s: AnyStr) -> AnyStr: ...
+@overload
+def filter(function: None,
+           t: Tuple[Any, ...]) -> Tuple[Any, ...]: ...
```

Comment:
Should be `Tuple[_T, ...]` in both the argument and return types.

Reply:
Actually, maybe this one should be `Tuple[Optional[_T], ...]` in the argument type and `Tuple[_T, ...]` in the return type, like in the second to last overload below.